spec/menu_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'menu'
+
+describe Menu do 
```

Comment:
It tests state rather then behaviour but I can't think of how you could change that one.